app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError)
```

Comment:
```suggestion
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```
or
```suggestion
from app.errors import (NotVaccinatedError,
                                         OutdatedVaccineError,
                                         NotWearingMaskError)
```
you can use `black` for style fixes. just install it in the terminal: `pip install black` and use it for your file/files/projects: `black app/cafe.py`